vins_estimator/src/estimator_node.cpp

```diff
@@ -11,38 +11,8 @@
 #include "estimator.h"
 #include "parameters_ros.h"
 #include "utility/visualization.h"
-
+#include <memory>
```

Comment:
Order of includes should follow the following order: 
![image](https://github.com/TanPinDa/VINS-Mono/assets/67696057/79029e2b-fb39-4a6d-8ef1-cff10a5fbf81)

Therefore for this particular file: 
```
#include <stdio.h>

#include <condition_variable>
#include <map>
#include <memory>
#include <mutex>
#include <queue>
#include <thread>

#include <cv_bridge/cv_bridge.h>
#include <opencv2/opencv.hpp>
#include <ros/ros.h>

#include "estimator.h"
#include "parameters_ros.h"
#include "utility/visualization.h"

```
source: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Thank you!